tools/configure_server_settings.rb

```diff
@@ -0,0 +1,37 @@
+if ARGV.empty?
+  puts "USAGE: #{__FILE__} server_id settings_path_separated_by_a_/ value [settings_path_separated_by_a_/ value]"
```

Comment:
I wonder if it makes sense to have a `--dry-run` option to report the server id, name, and existing value but don't make any changes.  Maybe for a followup PR.